src/Observable.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Observable.h
+ *
+ *  Created on: 30.10.2017
+ *      Author: abx852
+ */
+
+#ifndef OBSERVABLE_H_
+#define OBSERVABLE_H_
+
+#include <vector>
+#include "Observer.h"
+
+class Observable {
+public:
+	void register_observer(Observer* o) {
+		observers.push_back(o);
```

Comment:
hier noch eine Abfrage, ob der Observer eventuell schon in dem Vector enthalten war. Wenn ja, dann Observer nicht hinzufügen und  ERROR_LOG<<"tried to add Observer for second time"

Reply:
done